js/viz/graphic_objects.d.ts

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
If these new functions are used with all charts, they should be moved to the `common/charts` module